server/src/routes/register.js

```diff
@@ -1,10 +1,11 @@
 const mysql = require('../lib/mysql')
 const sessions = require('../lib/db/sessions')
-const bcrypt = require('bcrypt')
+const bcrypt = require('bcryptjs')
 
 const emailRegexp = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/
 const alphanumericRegexp = /^[a-z0-9]+$/i
 const initialMoney = 450000
+const initialUpdateDate = Math.floor(Date.now() / 1000)
```

Comment:
```suggestion
const initialMoneyUpdateDate = Math.floor(Date.now() / 1000)
```

Reply:
Esto debería estar en la propia requesr porque si se inicializa aquí siempre va a ser el timestamp de cuando se inició el servidor

Reply:
Fixed in 4abb908224f219791b0b567b3f697120186ada3d